### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by ilovesummer**
Please, review the changes.

### CODE REVIEW

**Comment by unsaniba** [comment]
Gotcha

**Comment by nchullip** [comment]
This looks good

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # git_lesson
 
 This is a readme file
+
+This line is from the feature1 branch.
+
+Hello reviewer.
```
Comment (ilovesummer) [inline README.md:7]: Why do we need this message?

**Comment by nchullip** [comment]
Can be merged to main

**Comment by michaelrenner20** [comment]
LGTM

**Comment by calchui** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (unsaniba) [inline README.md:7]: looks good

**Review by unsaniba — APPROVED** [review]

**Comment by michaelrenner20** [comment]
Time for dinner!

**Comment by NiyiOmidire** [comment]
LGTM

**Pull request merged**